keras_cv/models/object_detection/retina_net/__internal__/layers/prediction_head.py

```diff
@@ -64,9 +64,22 @@ def call(self, x, training=False):
 
     def get_config(self):
         config = {
-            "bias_initializer": self.bias_initializer,
+            "bias_initializer": keras.initializers.serialize(
+                self.bias_initializer
+            ),
             "output_filters": self.output_filters,
             "num_conv_layers": self.num_conv_layers,
         }
         base_config = super().get_config()
         return dict(list(base_config.items()) + list(config.items()))
+
+    @classmethod
+    def from_config(cls, config):
+        config.update(
+            {
+                "bias_initializer": keras.initializers.deserialize(
+                    config["bias_initializer"]
+                )
+            }
+        )
+        return super().from_config(config)
```

Comment:
@nkovela1 Would you please take a look if I did something weird here? Is there a way to implicitly serialize and deserialize the initializer? Thanks.

Reply:
Hi Haifeng,

I believe the explicit serialization and deserialization of the bias initializer is unnecessary here. It should already be taken care of unless you are using a custom (non-Keras) initializer. Please let me know if removing serialization there results in any errors.

Reply:
We will stick to the explicit serialization for initializers for now : )
Thanks, @nkovela1!